hack/generator/pkg/astmodel/struct_definition_test.go

```diff
@@ -18,11 +18,11 @@ func Test_NewStructDefinition_GivenValues_InitializesFields(t *testing.T) {
 	const group = "group"
 	const version = "2020-01-01"
 	fullNameField := createStringField("fullName", "Full legal name")
-	familyNameField := createStringField("familiyName", "Shared family name")
+	familyNameField := createStringField("familyName", "Shared family name")
 	knownAsField := createStringField("knownAs", "Commonly known as")
 
 	ref := NewTypeName(*NewLocalPackageReference(group, version), name)
-	definition := NewStructDefinition(ref, NewStructType(fullNameField, familyNameField, knownAsField), false)
+	definition := NewStructDefinition(ref, NewStructType().WithFields(fullNameField, familyNameField, knownAsField), false)
```

Comment:
minor/style: Wondering at what point it makes sense to break this on to multiple lines. Might not be there quite yet but seems to be getting close.

Reply:
Every time I've tried to break Go code across multiple lines, the compiler barks at me. #compilersaysno